src/app/strategies/page.tsx

```diff
@@ -62,7 +62,7 @@ const strayegyList: React.FC = () => {
     <div className='container'>
         <section className='section'>
         <div className='box'>
-            <a href='/newStrategy' className='tabs-more' > New Strategy </a>
+            <a href='/strategy/newStrategy' className='tabs-more' > New Strategy </a>
```

Comment:
Simplify route names, which should be simple, direct, convenient, and easy to remember.